t/lib/Test/MockOnfido.pm

```diff
@@ -0,0 +1,20 @@
+package Test::MockOnfido;
+use Mojolicious::Lite;
+use IO::Async::Loop;
+use IO::Async::Timer::Absolute;
+
+my $loop = IO::Async::Loop->new;
+
+# Wait 3 seconds before rendering a response
+get '/' => sub {
+    my $self = shift;
+    $loop->add(IO::Async::Timer::Absolute->new(
+                                               time      => time + 3,
+                                               on_expire => sub { $self->render(text => 'Delayed by 3 seconds!') }
+                                              ));
+};
+
+use Test::Mojo;
+my $t = Test::Mojo->new;
+$t->get_ok('/');
+done_testing();
```

Comment:
What's the purpose of this file? Can you add some comments in it?

Reply:
Sorry, these 2 files are unused and I forget to remove them. I deleted them now